libs/common/src/models/domain/organization.ts

```diff
@@ -201,4 +203,11 @@ export class Organization {
   get hasProvider() {
     return this.providerId != null || this.providerName != null;
   }
+
+  static fromJSON(json: Jsonify<Organization>) {
```

Comment:
Missing an early return in case the json is null

```suggestion
  static fromJSON(json: Jsonify<Organization>) {
  if (json == null) {
      return null;
    }
```